playbook-website/app/views/pages/visual_guidelines_partials/_max_width.html.erb

```diff
@@ -19,6 +19,7 @@
     <%= pb_rails("section_separator", props: { orientation: "vertical", margin_x: "sm" }) %>
     <%= pb_rails("flex/flex_item", props: { lex: "1" }) do %>
       <%= pb_rails("caption", props: {text: "Values", margin_bottom: "sm"}) %>
+      <%= pb_rails("pill", props: { text: "xs", variant: "warning" }) %>
       <%= pb_rails("pill", props: { text: "sm", variant: "warning" }) %>
       <%= pb_rails("pill", props: { text: "md", variant: "warning" }) %>
       <%= pb_rails("pill", props: { text: "lg", variant: "warning" }) %>
```

Comment:
Sorry, I meant let's leave them (None, & 0) out from the visual guide. We WANT them documented as props